### PR TITLE
OPL-52: changes required to run 'new batch run design'

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OPL-52
- hotfix to adding default values for type and subtype
- changes to support 'work_data' from new batch run design

### CODE REVIEW

File: calcrule_third_party_payment/signals.py [hunk 1]
```diff
@@ -6,6 +7,7 @@
 def bind_service_signals():
     bind_service_signal(
         'signal_after_claim_batch_module_process_batch_service',
-        ThirdPartyPaymentCalculationRule.convert_batch,
+        ThirdPartyPaymentCalculationRule.convert_batch_old,
```
Comment (delcroip) [inline calcrule_third_party_payment/signals.py]: why did you change it to old ?

Comment (sniedzielski) [reply calcrule_third_party_payment/signals.py]: because this was the old approach. This signal is not used in that approach (from develop branch in claim batch etc)

Comment (sniedzielski) [reply calcrule_third_party_payment/signals.py]: Anyway this file is commented so as to return to previous solution in case of issues. But I can remove this if you prefer.

Comment (sniedzielski) [reply calcrule_third_party_payment/signals.py]: Moreover I changed this 'convert batch' method because of using this 'work_data' from new batch run instead of taking kwargs of batch run execution like: audit_user_id, location_id, period, year

Comment (sniedzielski) [reply calcrule_third_party_payment/signals.py]: @delcroip I removed old code. Now I think should be more readable.

**Review by delcroip — APPROVED** [review]
